cpp-data-aggregation/aggregation_producer.cpp

```diff
@@ -94,54 +89,55 @@ void run_aggregation_production(size_t n_bytes,
 
 int main(int argc, char* argv[]) {
 
-    std::string context("Data Aggregation Scaling Tests");
-
     MPI_Init(&argc, &argv);
 
     //initializing rank
     int rank;
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+    std::string context("Data Aggregation Tests Producer Rank " + std::to_string(rank));
```

Comment:
Maybe put a colon to separate it out, something like:
`Data Aggregation Tests: Producer Rank 15`